src/nl/surf/eduhub_rio_mapper/rio/loader.clj

```diff
@@ -73,8 +73,12 @@
                    (single-xml-unwrapper element "ns2:aangebodenOpleidingCode"))]
       (log-rio-action-response (str "SUCCESSFUL RESOLVE:" code) element)
       code)
-    (let [id (-> element xml-utils/element->edn :opvragen_rioIdentificatiecode_response :foutmelding :sleutelgegeven :sleutelwaarde)]
-      (log-rio-action-response (str "RESOLVE FAILED - ID: " id) element)
+    (let [foutmelding (-> element xml-utils/element->edn :opvragen_rioIdentificatiecode_response :foutmelding)
+          id (-> foutmelding :sleutelgegeven :sleutelwaarde)]
+      (when-not (= "A01161" (:foutcode foutmelding))
```

Comment:
Extract to constant, properly describe error -- "De externe identificatie komt niet voor in RIO" -- and reason to handle it differently.